### ISSUE TITLE
[3rd Party Mod Related] Bug: Reloaded randomly crashes when playing Persona 4 Golden

### ISSUE THREAD

**Issue by PictoNoMii** [Reloaded-Project/Reloaded-II#258]
[OldLogs.zip](https://github.com/Reloaded-Project/Reloaded-II/files/12775644/OldLogs.zip)
Both Reloaded and P4G randomly close while running. This only happens when I play with mods, and not when I launch via Steam

**Comment by AnimatedSwine37** [comment]
I helped Picto with this a bit on the CEP discord server and found that the issue is coming from a D3D call (not sure what function exactly).
If said call returns an error code the game calls DebugBreak which just crashes the game if you aren't using a debugger (very nice error handling :D)

Anyway, they've updated their drivers and hopefully that'll fix the problem. In case it doesn't I got them to enable crash dumps in the hopes that they're more helpful (we could at least get the error code I think then). It's unclear if this is actually a problem with Reloaded, we'll see...
